tgui/packages/tgui/interfaces/PreferencesMenu/preferences/features/base.tsx

```diff
@@ -238,7 +238,7 @@ export function FeatureValueInput(props: FeatureValueInputProps) {
     shrink: props.shrink,
     handleSetValue: changeValue,
     value: predictedValue,
-    act: act, // iris edit
+    act: act,
```

Comment:
```suggestion
    act,
```
What if we just forwarded

Reply:
I did give that a try with the validation. including act as "any". I'll give it a commit and see what happens